spring-cloud-dataflow-docs/src/main/asciidoc/howto.adoc

```diff
@@ -208,7 +208,7 @@ The target Spring Cloud Stream application starter should be re-packaged to supp
 The JDBC-sink can be used to insert message payload data into a relational database table. By default,
 it inserts the entire payload into a table named after `spring.application.name` value set at the
```

Comment:
checking the code and testing this, it seems that it isn't using the `spring.application.name` after all. It's either `messages` or what is specified for `--jdbc.table-name`